general/server/plugins/index.md

```diff
@@ -39,7 +37,22 @@ Plugin settings should be retained if you do not delete the `.xml` files from th
 
 ### Manual
 
-All plugins hosted on the repository can be built from source and manually added to your server as well. They just need to be placed in the plugin directory, which is something like `var/lib/jellyfin/plugins` on most Linux distributions. Once the server is restarted any additions should automatically show up in your list of installed plugins. If you can't see the new plugin there may be a file permission issue.
+All plugins hosted on the repository can be built from source and manually added to your server as well. They just need to be placed in the plugin directory, which is something like `/var/lib/jellyfin/plugins` on most Linux distributions. **Make sure to create a subdirectory for each plugin.** Once the server is restarted, any additions should automatically show up in your list of installed plugins.
+
+#### Troubleshooting plugins
+
+If you can't see the new plugin in the dashboard, then the first place to look should be the `meta.json` which Jellyfin automatically creates in the plugin folder.
+If you cannot locate this file, then check the file and folder permissions to ensure that Jellyfin has access to the plugin folder.
+
+The value in the `status` field of the meta.json file points to what is happening with the plugin.  
+
+* Restart - The plugin requires Jellyfin to be restarted.
```

Comment:
I believe other lists like this use colons instead of dashes in our documentation.